langchain/src/chains/base.ts

```diff
@@ -29,16 +29,30 @@ export type SerializedBaseChain = ReturnType<
 
 export interface ChainInputs {
   memory?: BaseMemory;
+  verbose?: boolean;
+  callbackManager?: CallbackManager;
 }
 
+const getVerbosity = () => false;
+
 /**
  * Base interface that all chains must implement.
  */
 export abstract class BaseChain implements ChainInputs {
   memory?: BaseMemory;
 
-  constructor(memory?: BaseMemory) {
+  verbose: boolean;
+
+  callbackManager: CallbackManager;
+
+  constructor(
+    memory?: BaseMemory,
+    verbose?: boolean,
+    callbackManager?: CallbackManager
```

Comment:
This seemed like the most bw-compat option, although ideally, we should do what @nfcampos did with `BaseLanguageModelParams`

Reply:
This is an abstract class, so the only people we'd be breaking by changing these args would be someone who has created a custom subclass, not sure how likely that is